trade-finance-manager-api/src/v1/controllers/deal.tasks.api-test.js

```diff
@@ -41,16 +41,52 @@ describe('createDealTasks', () => {
   });
 
   describe('shouldCreatePartiesTask', () => {
-    describe('when a deal has tfm.exporter.partyUrn', () => {
+    describe('when a deal has tfm.exporter.partyUrn but NO tfm.buyer.partyUrn and deal is BSS', () => {
+      it('should return true', () => {
```

Comment:
Please add test coverage for both `exporter` and `buyer` having party URN.

Reply:
done